### DISCUSSION TITLE
Component Library recommandation

### DISCUSSION THREAD

**Discussion by jaytonic** [tailwindlabs/tailwindcss#9321] [Help]
Hi,

I've started to build a website with Tailwind CSS, and so far I love it. I have been using PrimeFlex up until now, so there was a lot of similar points.

One things tough: I start to have the needs of some component with some logic, like time picker, date picker, toggle, grid with sort, toasts, confirm dialogs. 

Some of them are not-so-hard to implement, some of them are quite some work, but it feels that implementing all of them would be a bit like re-inventing the wheel.

I've been looking for a UI component library that integrates well with tailwind, but so far I've not been succesful in my research, especially when date/time picker comes into the equation.

I'm sure some other talented developers already had the same issue, do you mind sharing what you have been using?

**Comment by hoetmaaiers** [comment]
No answers followed... I came here with the same thoughts.
Do you have anything to share since you posted your question?

**Comment by severinlandolt** [comment]
Hey, I am a maintainer over at [Tremor](https://www.tremor.so), we integrate with Tailwind and also have a datepicker available. Happy coding :)

**Comment by hoetmaaiers** [comment]
Thanks @severinlandolt for the input, Tremor looks interesting (I am experienced with MUI). But I was looking into Tailwind in combination with "raw html" and Phoenix Liveview...

**Comment by bjufre** [comment]
@hoetmaaiers @jaytonic have you seen https://flowbite.com/? It pretty much has integration with Phoenix, Rails, Laravel, etc..
